### ISSUE TITLE
Epoch number and warmup epoch number

### ISSUE THREAD

**Issue by Huiimin5** [facebookresearch/mae_st#9]
Dear authors,

In Pretrain.md (https://github.com/facebookresearch/mae_st/blob/main/PRETRAIN.md), epochs and warmup_epochs are specified as 100 and 5, respectively, which seems inconsistent with configurations in your paper. With sampling_rate=4, I should set epochs to 200 in order to train an 800ep model, right? 

In Table 5, the warmup epoch is set to 120. But in Pretrain.md, it is set to 5. Which one should I follow? When epoch is set to 200, how am I supposed to set warmup_epochs?

Many thanks
